upload-to-commons.patch

```diff
@@ -1,21 +0,0 @@
-diff --git a/app/src/main/java/fr/free/nrw/commons/CommonsApplication.java b/app/src/main/java/fr/free/nrw/commons/CommonsApplication.java
```

Comment:
Oh I see. In that case, I would mention that explicitly in the commit message (or commit title). "Fix Codacy Issues" sounds a bit indirect.